bin/spectatord_main.cc

```diff
@@ -17,31 +17,6 @@ auto GetSpectatorConfig() -> std::unique_ptr<spectator::Config>;
 using spectatord::GetLogger;
 using spectatord::Logger;
 
-auto get_port_number(spdlog::logger* logger, const char* str) -> int {
```

Comment:
The IDE does not report any uses of this function, not sure if it is magically invoked by Abseil or not.

Reply:
That function should be removed. Good catch. That function and usage were used prior to the introduction of the absl Flags library.